dotcom-rendering/src/web/components/Callout/MultiSelect.tsx

```diff
@@ -21,12 +23,14 @@ export const MultiSelect = ({
 	<div data-testid={`form-field-${formField.id}`}>
 		{multiple ? (
 			<CheckboxSelect
+				validationErrors={validationErrors && validationErrors}
```

Comment:
If the user has filled  in the form correctly, validation errors may not exist, hence we use `&&`

Reply:
good call! One small suggestion for a refactor that might make the intention a bit more obvious to new readers of the codebase
```suggestion
				validationErrors={validationErrors ?? undefined}
```

Reply:
~although actually I'm seeing a typescript error locally for this file, are you getting that too?~

~I'm wondering if `[key in string]: string` in the type signature for `CheckboxSelect` should be `[key: string]: string` instead?~

**Update:** ignore me, I think it was a local issue with my tsc setup (plus I wasn't familiar with the `key in string` notation).